aioshelly/rpc_device/device.py

```diff
@@ -249,10 +252,19 @@ async def trigger_reboot(self, delay_ms: int = 1000) -> None:
     async def update_status(self) -> None:
         """Get device status from 'Shelly.GetStatus'."""
         self._status = await self.call_rpc("Shelly.GetStatus")
+        self._status.update(
+            {
+                item["key"]: {"value": item["status"].get("value")}
+                for item in self._dynamic_components
+            }
+        )
 
     async def update_config(self) -> None:
         """Get device config from 'Shelly.GetConfig'."""
         self._config = await self.call_rpc("Shelly.GetConfig")
+        self._config.update(
+            {item["key"]: item["config"] for item in self._dynamic_components}
+        )
```

Comment:
Would be cleaner if `get_dynamic_components` will do this so we have all in a single place when we get the new data